tensorrt_llm/executor/result.py

```diff
@@ -224,8 +244,10 @@ def _handle_sequence(self, finish_reasons, response_tensors,
                     f"Unknown finish reason: {finish_reasons[src_idx]}")
 
     @nvtx_range("handle_response", color="red", category="GenerationResultBase")
-    def _handle_response(self, response: Union["PostprocWorker.Output",
-                                               tllm.Response, ErrorResponse]):
+    def _handle_response(self,
+                         response: Union["PostprocWorker.Output", tllm.Response,
+                                         ErrorResponse],
+                         additional_outputs: Optional[Dict[str, Any]] = None):
```

Comment:
It seems that the addtional_outputs contains multiple members, and Dict is an opaque compared to a concrete type. How about making it a namedtuple:
```python
class AdditonalOutputs(Namedtuple):
    ''' some comments '''
    prompt_logprobs: Optional[...] = None
    generation_logprobs: Optional[...] = None
```

This can make this argument more intuitive and good for jumping in IDE as well.